app/helpers/hosts_helper.rb

```diff
@@ -247,7 +247,7 @@ def reports_show
 
   def name_field(host)
     return if host.name.blank?
-    host.managed? ? host.shortname : host.name
```

Comment:
I don't think we need to touch this. Simple `host_name_el.data('managed') === false` removal will do.

Reply:
Thank you for your review!
You are right - this removal was not necessary. However, I want to let you know I made the change so that when editing the host, the name of the managed and unmanaged host is just the shortname - now it is shortname + domain when the host is unmanaged. But also, I am not sure in which form it should be, and it might be resolved after https://github.com/theforeman/foreman/pull/9613 that you mentioned.